MainWindow.cpp

```diff
@@ -302,7 +337,7 @@ void MainWindow::openFile(QString fName) {
 
   MainWindow::setWindowTitle(fileInfo.fileName() + " - ENIGMA");
   _recentFiles->prependFile(fName);
-  openProject(std::unique_ptr<buffers::Project>(loadedProject));
+  openProject(std::move(loadedProject));

```

Comment:
Should document somewhere, maybe in the header, that `openProject` takes ownership of the buffer.